server/filters/convert-image-uri.js

```diff
@@ -57,7 +57,15 @@ function convertPathToIiifUri(originalUriPath, iiifRoot, size) {
   return `${iiifRoot}${originalUriPath}/full/${size}${isFullSize ? '' : ','}/0/default.${format}`;
 }
 
-export default function convertImageUri(originalUri, requiredSize, useIiifOrigin) {
+export function convertIiifUriToInfoUri(originalUriPath) {
+  if (originalUriPath.startsWith('https://iiif')) {
```

Comment:
Do we (polyfil / are happy not to) this?

Reply:
This isn't browser code